bukkit/src/main/java/org/popcraft/bolt/matcher/block/MossCarpetMatcher.java

```diff
@@ -1,20 +1,32 @@
 package org.popcraft.bolt.matcher.block;
 
+import com.google.common.collect.Sets;
 import org.bukkit.Material;
 import org.bukkit.block.Block;
 import org.bukkit.block.BlockFace;
 import org.bukkit.entity.EntityType;
 import org.popcraft.bolt.matcher.Match;
+import org.popcraft.bolt.util.EnumUtil;
 
 import java.util.Collections;
 import java.util.Set;
 
 public class MossCarpetMatcher implements BlockMatcher {
+    // Future: make immutable (Set.of)
+    private static final Set<Material> MOSS_CARPETS = Sets.newHashSet(Material.MOSS_CARPET);
```

Comment:
In general prefer to use `java.util` over anything in transitive dependencies like `com.google.common.collect` (even though yes, it is extremely unlikely this will be removed from the server).
```suggestion
    private static final Set<Material> MOSS_CARPETS = new HashSet<>(Set.of(Material.MOSS_CARPET));
```